api/src/users/controllers/UserController.ts

```diff
@@ -18,8 +16,14 @@ interface AuthenticatedRequest extends Request<object, object, UpdateUserBody> {
   };
 }
 
-export default {
-  getUser: (req: AuthenticatedRequest, res: Response) => {
+export default class UserController {
+  private userService: typeof UserService;
```

Comment:
This `typeof` definition needs to be changed to an imported `interface` definition. Importing the service class itself still binds controllers to these specific classes, though we want to be able to switch out the dependency when needed.